x/subspaces/client/cli/cli_test.go

```diff
@@ -195,7 +195,7 @@ func (s *IntegrationTestSuite) TestCmdQuerySubspace() {
 					"cosmos1s0he0z3g92zwsxdj83h0ky9w463sx7gq9mqtgn",
 					"cosmos1s0he0z3g92zwsxdj83h0ky9w463sx7gq9mqtgn",
 					time.Date(2020, 1, 1, 12, 00, 00, 000, time.UTC),
-				),
+				).SetAllowedFeeTokens(sdk.Coins{}),
```

Comment:
Why is this needed?

Reply:
It is added by mistake, removed.

Reply:
It is added to avoid from returning `nil` which actually expected to be `empty` slice.
It can be solved by adding new parameter to `NewSubspace`.